api/src/main/proto/stellarstation/api/v1/antenna/antenna.proto

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package stellarstation.api.v1.antenna;
```

Comment:
One thing I wondered about. The transmitter and receiver configuration are inside the monitoring proto. What's the reason for not also putting this message in the same proto?

Or maybe receiver and transmitter config could be considered antenna related and could be here? Not sure :)

Reply:
Yeah I had some trouble deciding what to try to make more generic and what's monitoring specific. Since an antenna is physical and can't really change, I figure it could be returned with passes, etc too while for radios, since they're configurable, in such cases we'd need to return something quite different listing "supported options".